reports/.gitignore

```diff
@@ -0,0 +1,4 @@
+# Ignore everything in this directory
```

Comment:
I would have a .keep file here and add the /reports/ directory to the main gitignore folder with an explicit exception for the .keep

Wonder what others opinions on this are.

Reply:
What is the value in keeping this directory in the first place? @benjaminhobbs

Reply:
@jtart 1. (from Paul) the npm script writes the report to this directory and without it existing the script fails. 2. I think in the future not ignoring this content would be a nice way to show lighthouse scores on each branch (or at least latet) in custom badges without having to rely on another service reporting back.

Reply:
I see, sucks about 1.

I agree with your first comment.